tsconfig.json

```diff
@@ -19,7 +19,8 @@
     ],
     "paths": {
       "@/*": ["./src/*"]
-    }
+    },
+    "types": ["vitest/globals", "vitest/jsdom"]
```

Comment:
Is there a doc that says we should do this?
because I don't it's what we would want, according to the Typescript Doc:
https://www.typescriptlang.org/tsconfig#types

Reply:
[In vitest doc](https://vitest.dev/config/#globals)~~~

Reply:
I see, I probably misunderstood what `types` does.
However I still think explicitness is better, what do you think?